StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Vertical Main Screen/AccessoryButton.swift

```diff
@@ -58,11 +59,14 @@ final class AccessoryButton: UIButton {
         super.init(frame: .zero)
         setTitle(accessoryType.text, for: .normal)
         setTitleColor(appearance.colors.primary, for: .normal) // TODO read secondary action color
-        titleLabel?.font = appearance.scaledFont(for: appearance.font.base.medium, style: .subheadline, maximumPointSize: 20)
+        titleLabel?.font = appearance.scaledFont(for: appearance.font.base.medium, style: .footnote, maximumPointSize: 20)
```

Comment:
This is technically font size 13 when the figma calls for 12. I felt like 12 just looks really small.